llvm/lib/Analysis/TargetLibraryInfo.cpp

```diff
@@ -562,12 +562,9 @@ static void initialize(TargetLibraryInfoImpl &TLI, const Triple &T,
     // or we have a reasonable detection strategy, we cannot use exp10 reliably
     // on Linux.
     //
-    // Fall through to disable all of them.
-    [[fallthrough]];
-  default:
-    TLI.setUnavailable(LibFunc_exp10);
-    TLI.setUnavailable(LibFunc_exp10f);
-    TLI.setUnavailable(LibFunc_exp10l);
+    TLI.setAvailableWithName(LibFunc_exp10, "__exp10");
+    TLI.setAvailableWithName(LibFunc_exp10f, "__exp10f");
```

Comment:
exp10l?

Is this a fully-covered switch? (no longer has a default case)

Reply:
This comment needs updating too

Reply:
Updated, no regressions present.

Reply:
The comment hasn't been updated?

Reply:
Yes, I can see the exp10l update in *Files changed*

Reply:
Ah apologies, have rectified it. Thanks!